libvips/foreign/vips2webp.c

```diff
@@ -348,7 +373,17 @@ write_webp_anim( VipsWebPWrite *write, VipsImage *image, int page_height )
 
 		WebPPictureFree( &pic );
 
-		timestamp_ms += delay;
+		int page_index = top / page_height;
```

Comment:
In-block declarations are a C99 feature and libvips is (still) C89. Could you move the declaration to the start of the for().